src/frequenz/sdk/timeseries/_ringbuffer.py

```diff
@@ -5,17 +5,19 @@
 
 from __future__ import annotations
 
+from collections.abc import Iterable
 from copy import deepcopy
 from dataclasses import dataclass
 from datetime import datetime, timedelta
-from typing import Any, Generic, List, Sequence, TypeVar
+from typing import Any, Generic, List, TypeVar
 
-import numpy as np
+import numpy.typing as npt
+from numpy import concatenate, float64
 from numpy.lib import math
 
 from frequenz.sdk.timeseries import Sample
```

Comment:
could you make this use relative paths for sdk components:

``` suggestion
from . import Sample
```

Reply:
I didn't know we can do that. sure, will update